resources/lang/id/forum.php

```diff
@@ -0,0 +1,268 @@
+            'to_0' => 'Hapus tanda "terkonfirmasi"',
+            'to_0_done' => 'Tanda "terkonfirmasi" telah dihapus',
+            'to_1' => 'Tambahkan tanda "terkonfirmasi"',
+            'to_1_done' => 'Tanda "terkonfirmasi" telah ditambahkan',
+        ],
+
+        'issue_tag_duplicate' => [
+            'to_0' => 'Hapus tanda "terselesaikan"',
+            'to_0_done' => 'Tanda "terselesaikan" telah dihapus',
+            'to_1' => 'Tambahkan tanda "terselesaikan"',
+            'to_1_done' => 'Tanda "terselesaikan" telah ditambahkan',
+        ],
+
+        'issue_tag_invalid' => [
+            'to_0' => 'Hapus tanda "tidak valid"',
+            'to_0_done' => 'Tanda "tidak valid" telah dihapus',
+            'to_1' => 'Tambahkan tanda "tidak valid"',
+            'to_1_done' => 'Tanda "tidak valid" telah ditambahkan',
+        ],
+
+        'issue_tag_resolved' => [
+            'to_0' => 'Hapus tanda "terselesaikan"',
+            'to_0_done' => 'Tanda "terselesaikan" telah dihapus',
+            'to_1' => 'Tambahkan tanda "terselesaikan"',
+            'to_1_done' => 'Tanda "terselesaikan" telah ditambahkan',
+        ],
+
+        'lock' => [
+            'is_locked' => 'Topik ini telah dikunci dan tidak dapat dibalas',
+            'to_0' => 'Buka topik',
+            'to_0_done' => 'Topik telah dibuka',
+            'to_1' => 'Kunci topik',
+            'to_1_done' => 'Topik telah dikunci',
+        ],
+
+        'moderate_move' => [
+            'title' => 'Pindahkan ke forum lain',
+        ],
+
+        'moderate_pin' => [
+            'to_0' => 'Lepas sematan topik',
+            'to_0_done' => 'Sematan topik telah dilepaskan',
+            'to_1' => 'Sematkan topik',
+            'to_1_done' => 'Topik telah disematkan',
+            'to_2' => 'Sematkan topik dan tandai sebagai pengumuman',
+            'to_2_done' => 'Topik telah disematkan dan ditandai sebagai pengumuman',
+        ],
+
+        'show' => [
+            'deleted-posts' => 'Post yang Dihapus',
+            'total_posts' => 'Jumlah Post',
+
+            'feature_vote' => [
+                'current' => 'Prioritas saat ini: +:count',
+                'do' => 'Promosikan permintaan ini',
+
+                'user' => [
+                    'count' => '{0} tidak ada suara|{1} :count suara|[2,*] :count suara',
+                    'current' => 'Anda memiliki :votes tersisa.',
+                    'not_enough' => 'Anda tidak memiliki cukup hak suara untuk dapat mempromosikan gagasan ini lebih jauh.',
+                ],
+            ],
+
+            'poll' => [
+                'vote' => 'Pemilihan',
+
+                'detail' => [
+                    'end_time' => 'Pemilihan akan berakhir pada :time',
+                    'ended' => 'Polling telah berakhir :time',
+                    'total' => 'Jumlah suara: :count',
+                ],
+            ],
+        ],
+
+        'watch' => [
+            'to_not_watching' => 'Tidak dimarkahi',
```

Comment:
Di `beatmapset_watches` pakainya "amati"...?

Reply:
I though it's different use, right?
In beatmapset_events it uses "watch" instead of "bookmark" like in the forum.